public_dropin_environments/triton_server/dr_requirements.txt

```diff
@@ -0,0 +1 @@
+requests
```

Comment:
Am I missing something, how does drum get installed, doesn't that need to be listed as a requirement because we aren't using the standard drop-in base env?

Also, I think we should freeze these deps so re-builds are deterministic.

Reply:
Drum should be listed here. I removed it during the local testing. 
Added the `datarobot-drum==1.10.20` which is the version without any Triton support, but we'll update it later.

Reply:
pined requests to `requests==2.31.0`.